test/testdata/golden/gapic/lang/csharp_gapic.yaml

```diff
@@ -0,0 +1,5 @@
+type: com.google.api.codegen.ConfigProto
```

Comment:
Not that I know of. I think we can either

(1) keep the static files checked into `artman`
(2) add GAPIC config for library to `googleapis` and submodule

I suspect (2) will be slightly easier to maintain, but up to you.